suite-native/graph/src/components/Graph.tsx

```diff
@@ -1,27 +1,53 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { LineGraph, GraphPoint } from 'react-native-graph';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box } from '@suite-native/atoms';
+import { Box, Text } from '@suite-native/atoms';
+import { getExtremaFromGraphPoints, sumLineGraphPoints } from '@suite-common/wallet-graph';
+
+import { AxisLabel } from './AxisLabel';
 
 type GraphProps = {
     points: GraphPoint[];
 };
 
 const graphWrapperStyle = prepareNativeStyle(_ => ({
-    width: '100%',
-    height: 150,
-    marginVertical: 40,
+    height: 250,
+    justifyContent: 'center',
+    alignItems: 'center',
 }));
 
 const graphStyle = prepareNativeStyle(_ => ({
     alignSelf: 'center',
-    aspectRatio: 21 / 9,
+    aspectRatio: 1.4,
+    width: '100%',
 }));
 
 export const Graph = ({ points = [] }: GraphProps) => {
     const { applyStyle } = useNativeStyles();
 
+    const nonZeroSumOfGraphPoints = useMemo(() => sumLineGraphPoints(points) > 0, [points]);
+    const extremaFromGraphPoints = useMemo(() => getExtremaFromGraphPoints(points), [points]);
+
+    const axisLabels = useMemo(() => {
```

Comment:
On the latest MVP screens, I see that the design of labels for points is different. 
<img src="https://user-images.githubusercontent.com/45338719/197535656-ebee8ff4-ac44-4ee3-82c1-37c500f7013a.png" width="400" />

Reply:
I would prefer to solve this as follow-up desing shoudn't be focus of this PR